akka-http/src/main/scala/akka/http/scaladsl/client/RequestBuilding.scala

```diff
@@ -18,37 +18,39 @@ import HttpMethods._
 trait RequestBuilding extends TransformerPipelineSupport {
   type RequestTransformer = HttpRequest ⇒ HttpRequest
 
+  implicit val timeout = Timeout(1.second)
+
   class RequestBuilder(val method: HttpMethod) {
     def apply(): HttpRequest =
       apply("/")
 
-    def apply(uri: String): HttpRequest =
+    def apply(uri: String)(implicit timeout: Timeout): HttpRequest =
       apply(uri, HttpEntity.Empty)
 
-    def apply[T](uri: String, content: T)(implicit m: ToEntityMarshaller[T], ec: ExecutionContext): HttpRequest =
+    def apply[T](uri: String, content: T)(implicit m: ToEntityMarshaller[T], ec: ExecutionContext, timeout: Timeout): HttpRequest =
```

Comment:
(It wasn't reported since this is in `akka-http`, which is experimental; `akka-http-core` isn't experimental)